sdk/helper/certutil/helpers.go

```diff
@@ -586,21 +594,32 @@ func ValidateSignatureLength(hashBits int) error {
 	return nil
 }
 
-func ValidateKeyTypeLength(keyType string, keyBits int) error {
+func ValidateKeyTypeLength(keyType string, keyBits *int) error {
+	if *keyBits == 0 {
+		newValue, present := defaultAlgorithmKeyBits[keyType]
+		if present {
+			*keyBits = newValue
+		}
+	}
+
+	if keyType == "rsa" && *keyBits < rsaMinimumSecureKeySize {
```

Comment:
nit: This could be moved into the default case below simplifying the conditional.

Reply:
I ended up not moving it into the default case as I wanted to ensure that, even if `rsaMinimum` is updated to say 3072, we'd block 2048 (which is a good/well-known RSA key size).